rust-bins/src/bin/keygen.rs

```diff
@@ -143,7 +138,7 @@ struct GenRand {
 #[structopt(
     about = "Tool for generating keys",
     author = "Concordium",
-    version = "0.2"
+    version = "1"
```

Comment:
When I do, e.g., `./keygen keygen-ar --version`, it returns `keygen-keygen-ar 0.1.0` . Is this intended?

Reply:
Fixed.